integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -402,3 +402,15 @@ def create_empty_df(spark, path):
         lambda spark, path: spark.read.parquet(path),
         data_path,
         conf=writer_confs)
+
+from spark_session import is_before_spark_330
+@pytest.mark.order(1) # at the head of xdist worker queue if pytest-order is installed
+@pytest.mark.skipif(is_before_spark_330(), reason='DayTimeIntervalGen is not supported before Spark 3.3.0')
```

Comment:
```suggestion
@pytest.mark.skipif(is_before_spark_330(), reason='DayTimeIntervalType is not supported before Spark 3.3.0')
```

Reply:
Updated: "DayTimeInterval is not supported before Pyspark 3.3.0"